common/app/layout/CollectionEmail.scala

```diff
@@ -55,7 +57,11 @@ object CollectionEmail {
 
 case class CollectionEmail(id: String, contentCollections: List[EmailContentContainer]) {
   def collections: List[EmailContainer] = {
-    val (start, end) = contentCollections.splitAt(3)
+    val contentCollectionsWithoutUndisplayableFlagshipContainers = contentCollections.filterNot(container =>
```

Comment:
I decided to filter out the flagship containers that can't be displayed here, so that the `index` used later on is actually correct given the containers displayed

Reply:
makes sense, though I'm wondering if it's possible to filter in `emailFrontBody.scala.html` to avoid leaking this 'flagship' stuff any further?